### PR TITLE
40/add new page

### PR DESCRIPTION
**Pull request by bennpearson**
### What is the context of this PR?
Sections can be added and edited. 

### How to review 
Create a survey and check pages can be added.
Check section titles, section descriptions and question titles can be edited

### Related Trello tasks
https://trello.com/c/HHkiluUf/40-add-new-page-with-title-8
https://trello.com/c/c9EFD4So/5-add-edit-question-5

### CODE REVIEW

File: src/pages/DesignQuestionnaire.js [hunk 1]
```diff
@@ -2,73 +2,190 @@ import React from "react";
...
+
+  &:focus {
+    border: none;
+  }
+  &::placeholder {
+    color: #a3a3a3;
+  }
+`;
+
+class PageSection extends React.Component {
```
Comment (hamishtaplin) [inline src/pages/DesignQuestionnaire.js]: Extract this class into it's own component file.

File: src/pages/DesignQuestionnaire.js [hunk 2]
```diff
@@ -2,73 +2,190 @@ import React from "react";
...
+  };
+
+  render() {
+    return (
+      <StyledPageSection
+        onFocus={this.handleFocus}
+        onBlur={this.handleBlur}
+        focussed={this.state.focussed}
+      >
+        {this.props.children}
```
Comment (hamishtaplin) [inline src/pages/DesignQuestionnaire.js]: This should pass down the `focussed` state to it's children by mapping that state onto their props. Look at the way `Field` works to do this.

**Comment by WickyNilliams** [comment]
[Add New Page with Title (8)](https://trello.com/c/HHkiluUf/40-add-new-page-with-title-8)

**Comment by WickyNilliams** [comment]
Closing this, as I got into merge hell trying to update with master (due to files being deleted/moved). Will push to a new branch

**Pull request closed without merging**